src/mcts/stoppers/legacy.cc

```diff
@@ -72,6 +72,7 @@ class LegacyTimeManager : public TimeManager {
             params.GetOrDefault<float>("midpoint-move", 51.5f)),
         time_curve_steepness_(params.GetOrDefault<float>("steepness", 7.0f)),
         spend_saved_time_(params.GetOrDefault<float>("immediate-use", 1.0f)),
+        first_move_bonus_(params.GetOrDefault<float>("first-move-bonus", 0.0f)),
```

Comment:
I think the default here should be between 1 and 2, to give some boost in very short openings. In #645 we settled to the equivalent of 0.6, but that was probably too conservative.

Reply:
Yeah, probably a good idea to set the default to something like that. 1.8 would be exactly the difference between bookplybonus 0.25 (default) and 0.4 (what we use).

Reply:
Sounds good.